python/lsst/daf/butler/remote_butler/server/handlers/_external.py

```diff
@@ -83,76 +85,60 @@ async def get_client_config() -> dict[str, Any]:
     return {"cls": "lsst.daf.butler.remote_butler.RemoteButler", "remote_butler": {"url": "<butlerRoot>"}}
 
 
-@external_router.get("/v1/universe", response_model=dict[str, Any])
-def get_dimension_universe(factory: Factory = Depends(factory_dependency)) -> dict[str, Any]:
+@external_router.get("/v1/universe")
+def get_dimension_universe(
+    factory: Factory = Depends(factory_dependency),
+) -> GetUniverseResponseModel:
     # Allow remote client to get dimensions definition.
     butler = factory.create_butler()
-    return butler.dimensions.dimensionConfig.toDict()
+    return GetUniverseResponseModel(universe=butler.dimensions.dimensionConfig.to_simple())
 
 
 @external_router.get(
-    "/v1/dataset_type/{dataset_type_name}",
+    "/v1/dataset_type",
```

Comment:
If we know that we are enforcing that a name has no slashes it would be really nice from an API perspective if we did allow this simple form for retrieving the dataset type information. It seems like a very natural interface for single dataset type with no wildcards returning the details.